src/main/java/com/otto/lab2/service/impl/ParsingServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.otto.lab2.service.impl;
+
+import com.otto.lab2.model.Sentence;
+import com.otto.lab2.service.ParsingService;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParsingServiceImpl implements ParsingService {
+
+    @Override
+    public List<Sentence> parseText(String input) {
+
+        String splitRegEx = "((?<=\\.)|(?<=!)|(?<=\\?))";
+
+        List<String> sentencesFromInput = Arrays.stream(input.split(splitRegEx)).
+                map(s -> s.trim()).collect(Collectors.toList());
+
+        List<Sentence> sentenceList = sentencesFromInput.stream().map(s -> {
+            Sentence sentence = new Sentence();
+            sentence.setContent(s);
+            String[] words = s.split("\\s");
```

Comment:
Better use OOP here :). Create class Word , add to Sentences. In case of extending your APP, it could be useful